src/module.ts

```diff
@@ -1910,9 +1911,27 @@ export class Module {
       body
     );
     binaryen._free(cArr);
+    this.setLocalNames(ret, localNames);
```

Comment:
This can be removed now, so the additional parameter can be avoided. Can just call `.setLocalNames` after calling `addFunction` now.